NuGet/ChakraCore.Debugger.nuspec

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+  <metadata>
+    <id>ChakraCore.Debugger</id>
+    <version>$version$</version>
+    <description>Debugging companion library for the ChakraCore JavaScript engine.</description>
+    <authors>Microsoft</authors>
+    <projectUrl>https://github.com/Microsoft/ChakraCore-Debugger</projectUrl>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+  </metadata>
+
+  <files>
+    <file src="..\lib\Debugger.Service\ChakraDebugService.h" target="include" />
+    <file src="..\lib\Debugger.ProtocolHandler\ChakraDebugProtocolHandler.h" target="include" />
```

Comment:
Taking a look at other packages it looks like they put their headers in `build\native\include` and their libraries in `lib\native\vc140\`.  I can't find any specific documentation and that seems to be more of an issue for targeting multiple platforms, but it might be good for consistency.

Reply:
There is no absolute rule for headers or LIB files.
If anything, the right subfolder would be `lib\natie\vc141`, for this package will be built with VS2017.

Please confirm whether to move forward with this update.

Reply:
Note, these layout conventions are of little consequence, for what defines how to consume them is the property definitions in the package's .TARGETS file.

Reply:
Fair enough, it's probably fine where it is.